verification/curator-service/api/src/controllers/access.ts

```diff
@@ -0,0 +1,33 @@
+import { curry } from 'ramda';
+
+const accessControl = (
+  config: Object,
+) => (
+  curry((config, req, res, next) => {
```

Comment:
Neat. Is this just to simplify installation? I.e., load the config in our app declaration, then `app.use` the config-loaded-function?

Reply:
Yes, and to avoid having a constructor and a class just to close over one parameter.